utils.pwsh/Setup-Target.ps1

```diff
@@ -39,8 +39,8 @@ function Setup-BuildParameters {
     $VisualStudioData = Find-VisualStudio
 
     $VisualStudioId = "Visual Studio {0} {1}" -f @(
-        ([System.Version] $VisualStudioData.Version).Major
-        ( $VisualStudioData.Name -split ' ')[3]
+        ([System.Version] $VisualStudioData.InstallationVersion).Major
```

Comment:
Does this actually need the type specified, or is it just good practice? Checking both of these gives me the same output:
```
$VisualStudioData.InstallationVersion.GetType()
([System.Version] $VisualStudioData.InstallationVersion).GetType()
```

Reply:
It's good practise, because the cast makes it a version even if it's just a string - without it we'd have to check for the type before accessing the `Member` variable to avoid an exception.

Reply:
I mean that it doesn't seem to materially change the type or output of this particular data returned from VSSetup, which is already a `Version`. By comparison, the type for `InstallationLocation` from the old method _was_ a string, so it was necessary there. I'm not super fussed about it, it just seemed unnecessary with the new method.
```
PS C:\> $VisualStudioData.InstallationVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
17     5      33530  505

PS C:\> $VisualStudioData.InstallationVersion.GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Version                                  System.Object

> > ([System.Version] $VisualStudioData.InstallationVersion)

Major  Minor  Build  Revision
-----  -----  -----  --------
17     5      33530  505

PS C:\> ([System.Version] $VisualStudioData.InstallationVersion).GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Version                                  System.Object

PS C:\> $VisualStudioData = Get-CimInstance MSFT_VSInstance
PS C:\> $VisualStudioData[0].InstallLocation
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community

PS C:\> $VisualStudioData[0].InstallLocation.GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     String                                   System.Object
```